wizard_agent/brain/utils/util_functions.py

```diff
@@ -145,9 +145,10 @@ def get_shortest_path(start, end, game_state):
                 continue  # skip if visited
 
             # generate heuristics
-            neighbour.dist_to_start = manhattan_distance(neighbour.position, start_node.position)
-            neighbour.dist_to_goal = manhattan_distance(neighbour.position, goal_node.position)
-            neighbour.total_cost = neighbour.dist_to_start + neighbour.dist_to_goal
+            if neighbour:
```

Comment:
When will this ever be null? Neighbour is always assigned a Node object at line 144

Reply:
not sure but there was a null from smartbomb --> get_shortest_path --> manhattan distance

Reply:
start node or goal node can be empty. A fix might be to return None if either of them is None